destreamer.ts

```diff
@@ -16,9 +16,14 @@ import axios from 'axios';
  */
 
 const argv = yargs.options({
-    videoUrls: { type: 'array', alias: 'videourls', demandOption: true },
     username: { type: 'string', demandOption: false },
-    outputDirectory: { type: 'string', alias: 'outputdirectory', default: 'videos' },
+    outputDirectory: { type: 'string', alias: 'o', default: 'videos' },
+    videoUrls: {
+        alias: "u",

```

Comment:
I would use -V as an alias here so that we can use -u for --username

Reply:
Done, "u" is now to username, "V" urls